src/index.html

```diff
@@ -7,6 +7,330 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="header__logo">
+          <a href="https://www.moyo.ua/">
+            <img src="./images/logo.png" alt="logo">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#">Apple</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Samsung</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Smartphones</a>
+            </li>
+  
+            <li class="nav__item">
+              <a 
+                class="nav__link" 
+                href="#"
+                data-qa="nav-hover"
+              >Laptops & computers</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Gadgets</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Tablets</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Photo</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <div class="main">
+      <div class="card" data-qa="card">
+        <img 
+          class="card__image"
+          src="./images/imac.jpeg" 
+          alt="imac"
+        >
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">
+          Product code: 195434
+        </p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+  
+          <p class="card__rating-reviews">
+            Reviews: 5
+          </p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a 
+        class="card__buy" 
+        href="#"
+        data-qa="card-hover"
+        >
```

Comment:
Fix code format everywhere
```suggestion
        <a 
          class="card__buy" 
          href="#"
          data-qa="card-hover"
        >
```